earth_enterprise/rpms/build.gradle

```diff
@@ -402,16 +402,84 @@ if (project.hasProperty('buildOpenGee')) {
     openGeeCommonRpm.dependsOn stageOpenGeeInstall
 }
 
-
-task openGeeCommonDeb (type: Deb) {
+task openGeeCommonDeb (type: GeeDeb, dependsOn: openGeeSharedFiles) {
     packageName = 'opengee-common'
+    release = "1.${rpmPlatformString}"
+    version = openGeeVersion
+    user = 'root'
+    permissionGroup = 'root'
+    packageGroup = 'Application/Productivity'
```

Comment:
If this sets the `section` field of the Debian package, it should probably have a value of `misc`.  I don't think there's a section called `Application/Productivity` in Debian.  You can have a look at the list of defined sections at https://www.debian.org/doc/debian-policy/#sections.